app/src/main/java/com/framgia/fsalon/screen/customerinfo/CustomerInfoAdapter.java

```diff
@@ -0,0 +1,43 @@
+package com.framgia.fsalon.screen.customerinfo;
+
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentPagerAdapter;
+import android.support.v4.view.ViewPager;
```

Comment:
ERROR: Unused import: android.support.v4.view.ViewPager. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports